docs/how-tos/trigger-sync.md

```diff
@@ -1,61 +1,93 @@
-2. Change the configuration of the data manager to enable selective sync.
+{{% expand "A running machine connected to the Viam app. Click to see instructions." %}}
 
-### Requirements
+{{% snippet "setup-both.md" %}}
 
-Before configuring your sensor, you must [create a machine](/cloud/machines/#add-a-new-machine) and you also need to:
+{{% /expand%}}
 
-1. Enable [data capture](/services/data/capture/).
-2. Enable [cloud sync](/services/data/cloud-sync/).
+{{< expand "Enable data capture and sync on your machine." >}}
 
-### Add sensor to determine when to sync
+Add the [data management](/services/data/) service:
 
-In this example, you will configure sync to only trigger during a specific time frame of the day using an existing module [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time).
-If you need to trigger sync based on a different condition, you need to create your own module and adjust the module logic accordingly.
-Additional examples are available in this [GitHub Repo](https://github.com/viam-labs/trigger-sync-examples-v2).
+On your machine's **CONFIGURE** tab, click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+
+Select the `data management / RDK` service and click **Create**.
+You can leave the default data sync interval of `0.1` minutes to sync every 6 seconds.
+Also leave both **Capturing** and **Syncing** toggles in the "on" position.
+
+{{< /expand >}}
+
+### Add sensor to determine when to sync
 
 To use [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time):
 
-1. Go to your machine's **CONFIGURE** page and click **Create component**.
-2. Then select the `sync-at-time:timesyncsensor` model from the [`sync-at-time` module](https://app.viam.com/module/naomi/sync-at-time).
-3. Click **Add module**, then enter a name or use the suggested name for your sensor and click **Create**.
-   The sensor will return true and enable sync when in a specified time frame.
-4. To configure your time frame, go to the new component panel and copy and paste the following attribute template into your sensor’s attributes field:
+{{< table >}}
+{{% tablestep %}}
+**1. Add the sensor to your machine**
 
-   {{< tabs >}}
-   {{% tab name="Config builder" %}}
+On your machine's **CONFIGURE** page, click the **+** button next to your machine part in the left menu.
+Select **Component**, then search for and select the `sync-at-time:timesyncsensor` model provided by the [`sync-at-time` module](https://app.viam.com/module/naomi/sync-at-time).
+
+Click **Add module**, then enter a name or use the suggested name for your sensor and click **Create**.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Configure your time frame**
+
+Go to the new component panel and copy and paste the following attribute template into your sensor’s attributes field:
+
+{{< tabs >}}
+{{% tab name="Config builder" %}}
```

Comment:
I think we can actually just remove json template and json example and just have the code box from config builder. That simplifies it

Reply:
I'm making it be just the attributes box from config builder, but with a template tab and an example tab